docs/src/details/main-modules/images/color_formats.rst

```diff
@@ -0,0 +1,202 @@
+.. include:: /include/external_links.txt
+
+.. _images_color_formats:
+
+=============
+Color Formats
+=============
+
+Built-in Input Color Formats
+****************************
+
+The following image pixel color formats have built-in rendering support:
```

Comment:
There isn't such a thing as built-in rendering support, but there are 
- image decoders
- and GPUs, each having different capabilities